.github/workflows/update-conda-lockfile.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 # What branch does this action run on?
 # - workflow_dispatch: Whatever branch it was run against.
-# - schedule: Always the same branch (will be dev or main)
+# - schedule: Always runs on main
```

Comment:
Oh yay I'm glad we won't have push workflow files into `dev` and again into `main` to get the changes.